zh, en: update BR doc about omitting the spec.from/to when using new version (#812)
pingcap/docs-tidb-operator: en/backup-to-aws-s3-using-br.md

Before you perform ad-hoc backup, AWS account permissions need to be granted. This section describes three methods to grant AWS account permissions.

> **Note:**
>

> > If TiDB Operator >= v1.1.7 && TiDB >= v4.0.8, the `tikv_gc_life_time` will be adjusted by BR automatically, you can omit the step that creating the secret stores the account and password needed to access the TiDB cluster.

Reviewer: ```suggestion
> If TiDB Operator >= v1.1.7 && TiDB >= v4.0.8, `tikv_gc_life_time` will be adjusted by BR automatically. You can omit the step that creates the secret which stores the account and password needed to access the TiDB cluster.
```

    After the backup, no matter whether the backup is successful or not, as long as the previous `tikv_gc_life_time` is smaller than `.spec.tikvGCLifeTime`, TiDB Operator will try to set `tikv_gc_life_time` to the previous value.

    In extreme cases, if TiDB Operator fails to access the database, TiDB Operator cannot automatically recover the value of `tikv_gc_life_time` and treats the backup as failed. At this time, you can view `tikv_gc_life_time` of the current TiDB cluster using the following statement:

    {{< copyable "sql" >}}

    ```sql
    select VARIABLE_NAME, VARIABLE_VALUE from mysql.tidb where VARIABLE_NAME like "tikv_gc_life_time";
    ```

    In the output of the command above, if the value of `tikv_gc_life_time` is still larger than expected (10m by default), it means TiDB Operator failed to automatically recover the value. Therefore, you need to set `tikv_gc_life_time` back to the previous value manually:

    {{< copyable "sql" >}}

    ```sql
    update mysql.tidb set VARIABLE_VALUE = '10m' where VARIABLE_NAME = 'tikv_gc_life_time';
    ```

    > **Note:**
    >

>     > If TiDB Operator >= v1.1.7 && TiDB >= v4.0.8, the `tikv_gc_life_time` will be adjusted by BR automatically, you can omit the `spec.tikvGCLifeTime`.

Reviewer: ```suggestion
    > If TiDB Operator >= v1.1.7 && TiDB >= v4.0.8, `tikv_gc_life_time` will be adjusted by BR automatically, so you can omit `spec.tikvGCLifeTime`.
```

    kubectl create secret generic ${secret_name} --namespace=${namespace} --from-file=tls.crt=${cert_path} --from-file=tls.key=${key_path} --from-file=ca.crt=${ca_path}
    ```

    > **Note:**
    >

>     > If TiDB Operator >= v1.1.7 && TiDB >= v4.0.8, the `tikv_gc_life_time` will be adjusted by BR automatically, you can omit the `spec.from`.

Reviewer: ```suggestion
    > If TiDB Operator >= v1.1.7 && TiDB >= v4.0.8, `tikv_gc_life_time` will be adjusted by BR automatically, so you can omit `spec.from`.
```
